### ISSUE TITLE
Rootless containers running through podman-compose cannot access host services

### ISSUE THREAD

**Issue by petasis** [containers/podman-compose#1036]
**Describe the bug**
I have just upgraded to Fedora 40, and this has caused a havoc in my rootless podman-compose containers, that were working under Fedora 39. The root of the problem is that host services (e.g. the HTTPS web server running on host) are no longer accessible.

```
$ podman-compose version
podman-compose version 1.2.0
podman version 5.2.2
```

**Environment:**
 - OS: Linux / Fedora 40
 - podman version: 5.2.2
 - podman compose version: 1.2.0

**Additional context**

I have read several bugs reports about podman switching to pasta, and I have tried several approaches proposed (switching to slirp4netns in containers.conf, using pasta with options --map-guest-addr, etc.)

However, these seem to work when I manually run a container with podman run, but not when I run the containers through podman-compose.

Is there a solution to make my containers work again and access host services from them, as before the upgrade to Fedora 40?

I have been looking for more than 3 days for a solution, without success. Under Fedora 39, it worked out of the box, I haven't done anything special. I was accessing the host through its DNS hostname entry.

**Comment by juhojama** [comment]
Confirmed. While using Podman Compose to run the Podman containers, networking between containers and from container to host worked just fine.

**Comment by petasis** [comment]
I am confused. What worked under podman-compose? It does not work for me, containers cannot communicate with any service running on the host.

**Comment by BobbyWibowo** [comment]
Did you ever manage to get this working?

**Comment by petasis** [comment]
Yes, but throwing away this pasta thing, and restoring the previous approach.

I created a file named `~/.config/containers/containers.conf` and put inside:
```
[network]
default_rootless_network_cmd = "slirp4netns"
```

Don't forget to reboot the machine (or at least the podman services).

My personal opinion is that developers have done a huge mistake by adopting pasta and inserting such an incompatible change.

Running rootless containers is the main feature to choose podman, and loosing access to the host is a very bad decision.
